src/title/TitleStream.js

```diff
@@ -91,7 +91,7 @@ const TitleStream = (props: Props) => {
             isMuted={!stream.in_home_view}
             isPrivate={stream.invite_only}
             color={color}
-            size={20}
+            size={24}
```

Comment:
This actually ends up looking kind of oversized to me, in your handy screenshot at https://github.com/zulip/zulip-mobile/pull/4683#issuecomment-823584357 -- like it's out of proportion with the text that comes after it.

I think perhaps it makes sense here to think of this as not our usual use of an icon. The reasoning in previous adjustments like this (like the commit before this one) is about icon buttons -- but this icon isn't a button, it's not something you can activate. Instead it's sort of a sigil attached to the stream name, and it wants to be in a size that corresponds to that text.

It looks like the existing code ends up using the same size here as the font size on the title. Perhaps just replace the literal 20 here with an explicit reference to that?